src/theory/bags/theory_bags.cpp

```diff
@@ -213,34 +221,20 @@ void TheoryBags::presolve() {}
 
 void TheoryBags::eqNotifyNewClass(TNode n)
 {
+  if (n.getType().isBag())
```

Comment:
I don't think you actually need this method, right?  This is doing moditifcations to the state which should be SAT-context-dependent, but it we are not using SAT-context-dependent data structures.  This could lead to unexpected behavior.
I would delete the body for now.

Reply:
Done. I missed this old code which has no effect because the solver state is reset in each postCheck method